hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/NodeStatusUpdaterImpl.java

```diff
@@ -1136,9 +1130,8 @@ public void verifyRMHeartbeatResponseForNodeAttributes(
         } else {
           // case where updated node attributes from NodeAttributesProvider
           // is sent to RM and RM rejected the attributes
-          LOG.error("NM node attributes {" + getPreviousValue()
-              + "} were not accepted by RM and message from RM : " + response
-              .getDiagnosticsMessage());
+          LOG.error("NM node attributes {{}} were not accepted by RM and message from RM : {}." ,
+              getPreviousValue(), response.getDiagnosticsMessage() );
```

Comment:
I am not sure about this ``{{}}``, give a check if it works they way you want, in general it is very confusing can explore using `[{}]` as well

Reply:
I agree with you, I will modify the code.